src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/DBCaseRepository.java

```diff
@@ -111,10 +111,10 @@ public void saveCountyCourtJudgment(
         String authorisation,
         Claim claim,
         CountyCourtJudgment countyCourtJudgment,
-        boolean issue
+        boolean isByAdmission
     ) {
         final String externalId = claim.getExternalId();
-        LocalDateTime ccjIssuedDate = issue ? nowInUTC() : null;
+        LocalDateTime ccjIssuedDate = isByAdmission ? nowInUTC() : null;
```

Comment:
`ccjIssuedDate` is not required any more. please use `countyCourtJudgmentRequestedAt`